api/pkg/cli/cmd/get/get.go

```diff
@@ -211,22 +134,30 @@ func examples(kind string) string {
 }
 
 func (opts *options) GetResourceInfo() (string, error) {
+
+	Opts := so.Options{
```

Comment:
should we rename this to `askOpts` or something better ?

Reply:
May be only `ask` ??

Reply:
I said `askOpts` just because we are settings the `Options` struct